pkg/cmd/pulumi/watch.go

```diff
@@ -143,6 +145,9 @@ func newWatchCmd() *cobra.Command {
 		}),
 	}
 
+	cmd.PersistentFlags().StringArrayVarP(
+		&pathArray, "path", "", []string{""},
```

Comment:
One additional question as I looked deeper at this: What exactly are the allowed paths?  Can they be files or only folders? (it looks like the implementation only supports folders?).  Should we support files as well?  Can they be relative or absolute?  (it looks like the implementation only supports relative folders?). I'm okay supporting only a subset of these initially - but if so, we should be clear on what is supported in the help text.  Ideally though, we would support relative/absolute and files/folders.

Reply:
Checked all of your comments - all sounds good.

Initially, relative folders/files were supported, but now we also support absolute folders/files.

I updated the command descriptions, so now it should be clear to the user.

Hope it's fine but let me know if further amends are needed.

Thanks! 🍻

Reply:
Just wanted to let you know that I'm patiently waiting for an additional review here.

I'm sure you're all super busy, so, whenever you get a chance to check this, it's cool. 🙂 

Thank you! 🍺